README.md

```diff
@@ -51,7 +51,13 @@ git clone https://github.com/nvim-lua/kickstart.nvim.git %userprofile%\AppData\L
 
 ### Post Installation
 
-Run the following command and then **you are ready to go**!
+Start Neovim
+
+```sh
+nvim
+```
+
+The `Lazy` plugin manager will start automatically on the first run and install the configured plugins - as can be seen in the introduction video. After the installation is complete you can press `q` to close the `Lazy` UI and **you are ready to go**! Next time you run nvim `Lazy` will no longer show up. If you would prefer to hide this step and run the plugin sync from the command line, you can use:
```

Comment:
nit. Maybe it's just me, but I'd keep the
> If you would prefer...

separated by a newline. Cause the preceding message seems to be complete.

Reply:
@dam9000 I agree with @ro0gr.  Would it be possible to add the newline they suggest? I'd sincerely appreciate it and THANK YOU for your contribution in any case!

Reply:
Done.